packages/server/src/automations/steps/updateRow.js

```diff
@@ -66,9 +66,9 @@ exports.run = async function ({ inputs, appId, emitter }) {
   }
   const tableId = inputs.row.tableId
 
-  // clear any falsy properties so that they aren't updated
+  // clear any null or empty string properties so that they aren't updated
   for (let propKey of Object.keys(inputs.row)) {
-    if (!inputs.row[propKey] || inputs.row[propKey] === "") {
+    if (inputs.row[propKey] === null || inputs.row[propKey] === "") {
```

Comment:
We can use the loose equality operator `==` instead of `===` here to check for both null and undefined at the same time as @melohagan has suggested. i.e:
```js
inputs.row[propKey] == null || inputs.row[propKey] === ""
```

Reply:
Nice! TIL.

Reply:
Thanks, @Rory-Powell for the advice!